donor-portal/src/pages/about-us/index.tsx

```diff
@@ -1,221 +1,181 @@
+              individuals and organizations alike in executing our fiduciary
+              responsibility to maximize the utility of your donations.
             </p>
           </div>
-          <div className="partner-section-card">
-            <a href="https://sefglobal.org/">
-              <img src="/assets/images/about-us/SEF_Logo.png" alt="" />
-            </a>
+          <div className="accountability-and-transparency">
+            <h2>Transparency</h2>
             <p>
-              Sustainable Education Foundation (SEF) helps local students
-              connect with industry experts and academics around the world for
-              mentoring and career opportunities. SEF also supports Sri Lankan
-              researchers by popularising their work and connecting them with
-              undergraduate researchers for collaborations.
+              Elixir&apos;s primary design goal is transparency. While the Sri
+              Lanka Red Cross will monitor the flow of aid packages and provide
+              daily status updates, we have also contracted the services of KPMG
+              Sri Lanka, to perform an end-to-end audit of the Elixir process
+              and finances. Our financial team will also publish the monthly
+              statement of accounts on the Elixir website. We are committed to
+              continuously improving our processes. Please contact us at
+              elixir@redcross.lk should you have any questions or feedback for
+              us.
             </p>
           </div>
-          <div className="partner-section-card">
-            <a href="https://longform.watchdog.team/about-us">
-              <img src="/assets/images/about-us/watchdog-logo.png" alt="" />
-            </a>
-            <p>
-              Watchdog is a multidisciplinary team of fact checkers,
-              journalists, researchers and software engineers. We hunt hoaxes
-              and misinformation, investigate matters of public welfare, and
-              build software tools that help operations like ours.
-            </p>
+        </div>
+
+        <div className="leadership-section">
+          <h2>Leadership Team</h2>
+          <div className="profile-card-container">
+            <div className="profile-card">
+              <a href="https://www.linkedin.com/in/helani?originalSubdomain=lk">
+                <h3>Dr. Mahesh Gunasekara</h3>
+                <p>Director General</p>
+                <p>Sri Lanka Red Cross Society</p>
+              </a>
+            </div>
+            <div className="profile-card">
+              <a href="https://www.linkedin.com/in/helani?originalSubdomain=lk">
+                <h3>Helani Galpaya</h3>
+                <p>Citizen Volunteer</p>
+              </a>
+            </div>
+            <div className="profile-card">
+              {/* TODO: Add linkedin link */}
+              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
Do we have to disable this?

Reply:
yes until we find the correct link. (We can't have "#" as the link because of the rule).